src/global/globalConfig.ts

```diff
@@ -36,16 +36,15 @@ export enum AppName {
   GENOME_BROWSER = 'Genome browser',
   SPECIES_SELECTOR = 'Species selector',
   CUSTOM_DOWNLOADS = 'Custom downloads',
-  ENTITY_VIEWER = 'Entity viewer'
+  ENTITY_VIEWER = 'Entity viewer',
+  TOOLS = 'Tools' // this is the name of a group of apps
 }
```

Comment:
It seems that the only purpose of this enum is to be used as a title of an app passed to the AppBar component. Perhaps consider removing it?

Reply:
Maybe the mapping should `BLAST = 'Blast'` we dont need an entry for tools isn't it?

Reply:
This is what the mockup currently shows. Could be Andrea's mistake; I didn't ask 🤷‍♂️ :

![image](https://user-images.githubusercontent.com/6834224/148803970-41c3464d-b616-4140-9999-991560ff7e90.png)

As far as I can tell, it is the only use of this enum. If so, we don't really need it that much at all.

Reply:
Is that launchbar icon representing tools or blast?

Reply:
The icon is representing blast.